### PR TITLE
Missing slash

### PR DESCRIPTION
**Pull request by deathly809**
It seems we are missing a slash on one of the endpoints.  This makes the endpoint unusable.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by deathly809** [comment]
@dsgouda A slash was missing, I also fixed the readme.md output folder to take the csharp-sdks-folder parameter.  It broke the SDK generation.  Should I fix all the other readme files where that is broken as well?

**Comment by dsgouda** [comment]
@deathly809 yes, please do, but make sure you regenerate the code for each of the sdks again.

**Comment by deathly809** [comment]
@dsgouda I have fixed the readmes and need this PR to merge before I can regenerate the SDKS which are having issues.

File: specification/azsadmin/resource-manager/keyvault/readme.md [hunk 1]
```diff
@@ -48,26 +48,6 @@ csharp:
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.AzureStack.Management.KeyVault.Admin
   payload-flattening-threshold: 1
-  output-folder: $(csharp-sdks-folder)/KeyVault/KeyVault.Admin/Generated
+  output-folder: $(csharp-sdks-folder)/Generated
   clear-output-folder: true
 ```
-
-## Python
```
Comment (dsgouda) [inline specification/azsadmin/resource-manager/keyvault/readme.md]: If I understand correctly, you don't intend to generate any python sdks here? (or generate python sdks with proper settings

**Review by dsgouda — COMMENTED** [review]
LGTM for the most part.
Please acknowledge the comment before we merge.

File: specification/azsadmin/resource-manager/keyvault/readme.md (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/keyvault/readme.md]: We only intend to support C# so that we can create PowerShell.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**